### PR TITLE
[Server] Pagination on manifest and build listing

### LINKED ISSUE #15: Pagination on lists

### PR DESCRIPTION
**Pull request by BourgoisMickael**
Closes #15

### CODE REVIEW

File: server/src/action/abstract.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ class AbstractAction {
       this.validate,
       this._handler.bind(this)
     ]
+
+    this.DEFAULT_ITEM_PER_PAGE = 15
```
Comment (Arignir) [inline server/src/action/abstract.js]: Would be nice to have this in a configuration file instead

File: server/src/controller/build.js [hunk 2]
```diff
@@ -136,9 +136,12 @@ class BuildController {
...
       })
       .orderBy(sort, direction)
-      .fetchAll()
+      .simplePaginate({
+        limit: pagination.perPage,
+        page: pagination.page
+      })
 
-    return builds.toJSON()
+    return builds
```
Comment (Arignir) [inline server/src/controller/build.js:144]: Why did you removed `.toJSON()`? Could it be a mistake?

**Review by Arignir — CHANGES_REQUESTED** [review]

File: server/src/controller/build.js (hunk 2 above)
Comment (BourgoisMickael) [reply server/src/controller/build.js:144]: It's not a mistake. `fetchAll` function returns a object `Collection` which has the function `toJSON()` to convert the result to json but `simplePaginate` returns a json response like this:
```javascript
{
  data: [
    { }, // object 1
    { },
    ...
  ],
 meta: {
    //pagination
  }

File: server/src/action/abstract.js (hunk 1 above)
Comment (BourgoisMickael) [reply server/src/action/abstract.js]: it has been moved

**Review by Arignir — APPROVED** [review]

**Pull request merged**